res/i18n/fr/labels.json

```diff
@@ -481,38 +481,38 @@
 		"enabled": "Activé",
 		"fade_in": "Fondu ouverture",
 		"fade_out": "Fondu fermeture",
-		"head_cut": "Écrêtage",
+		"head_cut": "Coupe du début",
```

Comment:
@sadko4u : could you please explain us a bit more about what it is/does from your dev point of view? I'm not sure to be a big fan of this translation proposal.

Reply:
It's for a IR begining, and "ecretage" is "clipping" in this "saturation" meaning.

Reply:
@sadko4u : can you please explain us what does this do exactly?